example/index.ios.js

```diff
@@ -1,33 +1,34 @@
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- * @flow
- */
-
-import React, { Component } from 'react';
+import React, { Component } from 'react'
```

Comment:
I believe we should have only index.js file, please remove all *.ios.js files where they aren't required. General module API should be same on Android and IOS platforms, so there should be only one entry point in example